src/@[scope]/[name].md.js

```diff
@@ -179,18 +179,23 @@ function reviver(key, value) {
       x: {axis: "top", grid: true},
       y: {domain: d3.sort(d3.sort(versions.filter((d) => d.downloads > 0), (d) => -d.downloads).slice(0, 10).map((d) => d.version), (a, b) => semverCompare(b, a))},
       marks: [
-        Plot.barX(versions.filter((d) => d.downloads > 0), {
+        Plot.barX(versions, {
```

Comment:
I wanted to omit versions without any downloads from the y-axis.

Reply:
the explicit domain filters them out